group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/GeneralUniformAssignmentBuilderTest.java

```diff
@@ -103,13 +114,13 @@ public void testTwoMembersSubscribedToNonexistentTopics() {
         members.put(memberA, new AssignmentMemberSpec(
             Optional.empty(),
             Optional.empty(),
-            Collections.singletonList(topic3Uuid),
+            new TopicIds(Collections.singleton(topic3Name), topicsImage),
```

Comment:
Why do we need this change? It looks like the previous version would just work, no?

Reply:
I removed all TopicIds initializations in the assignor tests.